pages/common/mpg321.md

```diff
@@ -0,0 +1,33 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> Mpg321 was written to be a drop-in replacement for the (by-then) non-free mpg123 player.
```

Comment:
Perhaps this will read better?
```suggestion
> Drop-in replacement for the (previously) non-free `mpg123`.
```

Reply:
Nope, I'd keep it (and even add some more historic info).